staging/src/k8s.io/apimachinery/pkg/api/resource/BUILD

```diff
@@ -46,7 +46,7 @@ go_library(
 go_test(
     name = "go_default_xtest",
     srcs = ["quantity_example_test.go"],
-    deps = ["//vendor/k8s.io/apimachinery/pkg/api/resource:go_default_library"],
+    deps = [":go_default_library"],
```

Comment:
These changes in staging look fishy but were generated by the script. I don't see why this should be changed because of bumping a dependency...?

Reply:
`update-bazel.sh` removes these changes. But `godep-save.sh` brings them back. :thinking: 

On the CI - `verify bazel` failed but `verify godeps` passed.

Ran `update-bazel.sh` and checking if `verify godeps` passes.